app/views/news/2016-11-30-november-2016-monthly-update.html.md

```diff
@@ -0,0 +1,87 @@
+
+We had three improvements with our meta tooling this month. The [how_is](https://github.com/how-is/how_is) tool got another [release](https://github.com/how-is/how_is/blob/v11.0.0/CHANGELOG.md#1100), and will soon be used to actively [appraise the status](https://how-is.github.io/how-is-rubygems/2016/12/01/report.html) of our various projects.
+
+We also created a [time-card](https://github.com/rubytogether/rubytogether-lita/blob/84f6bec5d2faabbcb914132cd2880b8e7bacf33e/lib/lita/handlers/time_card.rb) handler for the [lita](https://www.lita.io/) bot, soon to be available for use in a chat room near you!
+
+Finally, we shipped a new merge-bot: [@bundlerbot](https://github.com/bundlerbot). We've been using the [homu.io](http://homu.io) service for over a year, but a long-standing bug prevented us from managing team member permissions. Now we have [our own copy of homu](https://bundlerbot-homu.herokuapp.com), and we look forward to updating team members and merging only green code for years to come.
+
+Last in Ruby Together news, this month we partnered with [@mrb_bk](https://twitter.com/mrb_bk)'s [Computer Modern](http://computermodern.io). We need more companies to join as members for Ruby Together to be sustainable in the long run, so it's exciting to have expert help. We'll be working with Computer Modern in the coming weeks and months to create resources for Ruby developers and Ruby Together members.
+
+
+## bundler news
+
+This month bundler and [bundler.io](bundler.io) got an array of documentation fixes / updates, due in large part to our new bundler contributor [@colby-swandale](https://github.com/colby-swandale).
+
+Bundler got a [Request For Comments](https://github.com/bundler/rfcs) repo, inspired by the one utilized by [Rust](https://www.rust-lang.org/en-US/). [Fixed](https://github.com/bundler/bundler/pull/5176) outdated not listing all outdated gems. And [fixed](https://github.com/bundler/bundler/pull/5213) an `ObjectBoundsExceededError` with [Rubinius](https://rubinius.com/).
+
+As mentioned last month, Bundler 1.13 shipped with [`required_ruby_version` support](http://bundler.io/blog/2016/09/08/bundler-1-13.html) for Gemfiles containing a `ruby` declaration. Building on that, we hoped to release 1.14 with automatic support for the currently running Ruby version.
+
+While @segiddins [nobly attempted](https://github.com/bundler/bundler/pull/5013) to implement it, we ran into a plethora of edge cases and problems with backwards compatibility. We'll aim to post a longer writeup of the situation on [the Bundler blog](https://bundler.io/blog) soon.
+
+In the meantime, the workaround for now is to put `ruby RUBY_VERSION` in your `Gemfile`. Now that we've concluded that automatic Ruby version management needs to wait for Bundler 2.0, we'll be able to release 1.14 very soon.
```

Comment:
I don't think this is good advice, I'd prefer people put an explicit version in their Gemfiles